app/service/event_svc.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+import json
+import websockets
+
+from app.service.interfaces.i_event_svc import EventServiceInterface
+from app.utility.base_service import BaseService
+
+
+class EventService(EventServiceInterface, BaseService):
+
+    def __init__(self):
+        self.log = self.add_service('event_svc', self)
+        self.contact_svc = self.get_service('contact_svc')
+        self.ws_uri = 'ws://{}'.format(self.get_config('app.contact.websocket'))
+
+    async def observe_event(self, event, callback):
+        ws_contact = self.contact_svc.get_contact('websocket')
+        handle = _Handle(event, callback)
+        ws_contact.handler.handles.append(handle)
+
+    async def fire_event(self, event, **callback_kwargs):
+        uri = '{}/{}'.format(self.ws_uri, event)
```

Comment:
this isn't wrong however generally I would use url lib to construct the uri, helps prevent misformed Uris